ramls/item.json

```diff
@@ -54,6 +54,11 @@
       "description": "Text, with input likely validated by the barcode scanner",
       "type": "string"
     },
+    "effectiveShelvingOrder": {
+      "type": "string",
+      "description": "A system generated normalization of the call number that allows for call number sorting in reports and search results",
+      "readonly": true
+    },
```

Comment:
Please change the minor version of all interfaces that use this schema to reflect this new property

Reply:
Please also change the dependency on `item-storage` to use the interface version that includes the new property in storage.

Reply:
The only raml file I can find that directly references item.json and has versioning information is inventory.raml-I incremented that version to 10.9.  I also changed the versioning of that module in moduledescriptor-template to match.  I updated the reference to item-storage in moduledescriptor-template to match the one in mod-inventory-storage (8.9). I hope that's right?

Reply:
I think that's right